agent/util-scripts/pcpint/pcp-mapping.json

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
Can you use Python3's `json` module to load this and sort the keys alphabetically, using 4 space indentation?